mutable_tree.go

```diff
@@ -616,19 +616,33 @@ func (tree *MutableTree) LoadVersion(targetVersion int64) (int64, error) {
 // LoadVersionForOverwriting attempts to load a tree at a previously committed
 // version, or the latest version below it. Any versions greater than targetVersion will be deleted.
 func (tree *MutableTree) LoadVersionForOverwriting(targetVersion int64) (int64, error) {
-	latestVersion, err := tree.LoadVersion(targetVersion)
+	return tree.LoadVersionForOverwritingWithMode(targetVersion, false)
+}
+
+// LoadVersionForOverwritingWithMode call LoadVersionForOverwriting with offlineRollback
+// to allow rollback in a quick and dirty way.
+func (tree *MutableTree) LoadVersionForOverwritingWithMode(targetVersion int64, offlineRollback bool) (int64, error) {
+	var latestVersion int64
+	var err error
+	if !offlineRollback {
+		latestVersion, err = tree.LoadVersion(targetVersion)
+	} else {
+		latestVersion, err = tree.LazyLoadVersion(targetVersion)
+	}
 	if err != nil {
 		return latestVersion, err
 	}
 
-	if err = tree.ndb.DeleteVersionsFrom(targetVersion + 1); err != nil {
+	if err = tree.ndb.DeleteVersionsFrom(targetVersion+1, offlineRollback); err != nil {
 		return latestVersion, err
 	}
 
-	if !tree.skipFastStorageUpgrade {
+	if !tree.skipFastStorageUpgrade && !offlineRollback {
 		if err := tree.enableFastStorageAndCommitLocked(); err != nil {
 			return latestVersion, err
 		}
+	} else if err = tree.ndb.Commit(); err != nil {
```

Comment:
@yihuang seems we still need commit if `skipFastStorageUpgrade`?

Reply:
I see, so it don't commit previousely?

Reply:
yup, I wonder if need fix as a separate bug

Reply:
I see, the current rollback cmd maybe don't work at all if the fast node is disabled 😂

Reply:
Closing this PR, let's limit the options.